var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -37,92 +38,120 @@ class Neuron(Package):
     version('7.3', '993e539cb8bf102ca52e9fefd644ab61')
     version('7.2', '5486709b6366add932e3a6d141c4f7ad')
 
-    variant('binary',        default=False, description="Create special as a binary instead of shell script")
-    variant('coreneuron',    default=True,  description="Patch hh.mod for CoreNEURON compatibility")
-    variant('cross-compile', default=False, description='Build for cross-compile environment')
-    variant('debug',         default=False, description='Build debug with O0')
-    variant('mpi',           default=True,  description='Enable MPI parallelism')
-    variant('multisend',     default=True,  description="Enable multi-send spike exchange")
-    variant('profile',       default=False, description="Enable Tau profiling")
-    variant('python',        default=True,  description='Enable python')
-    variant('shared',        default=True,  description='Build shared libraries')
-    variant('pysetup',       default=True,  description="Build Python module with setup.py")
-    variant('rx3d',          default=False, description="Enable cython translated 3-d rxd. Depends on pysetup")
+    variant('cmake',                   default=False, description="Build NEURON using cmake")
```

Comment:
I'd prefer to drop the extra whitespace to align the arguments, will make future diffs smaller.

Reply:
I removed a large portion of them but kept some to look nicer, similar to what is done in other projects. Let me know what you think :+1: